### DISCUSSION TITLE
Unordered categorical auxiliary variable

### DISCUSSION THREAD

**Discussion by aloejhb** [AdaptiveMotorControlLab/CEBRA#61] [Q&A] [answered]
Hi,
thanks for the great work with CEBRA.

I have a question about using unordered categorical labels as auxiliary variable.

For example, I would like to label 0 to 5 seconds of my recordings as condition A, 6 to 10 seconds as condition B, 11-15 seconds as C, 16 to 20 seconds as B, and so on.

The distances between AB, BC, CA are considered equal.

In this case, is discrete index still applicable? It seems that discrete index cares about the ordering of the labels, i.e. A < B < C.

Could there be a workaround if that is the case?

Thanks a lot!

**Comment by stes** [comment, marked as answer]
Hi @aloejhb , thanks for your question.

Some clarification: 

> It seems that discrete index cares about the ordering of the labels

This is not the case. The discrete index only cares about "equality" between labels, but does not assume any ordering. That means that you can go ahead and use the discrete index in your setup.

It *might* be interesting for your case to additionally label the "time within the condition" in your setting, and pass this as additional (continuous) information.

**Comment by aloejhb** [comment]
Thank you a lot for the answer and suggestions! This solves my confusion.
